adapters/repos/db/vector/hnsw/priorityqueue/queue.go

```diff
@@ -95,6 +95,12 @@ func (l *Queue) Pop() Item {
 	return out
 }
 
+func (l *Queue) SubstitudeTop(id uint64, distance float32) {
```

Comment:
Do you mean `SubstituteTop`? Maybe `ReplaceTop` is a better name?

Reply:
Done